### PR TITLE
History api

### LINKED ISSUE #52: Add operations history API

### PR DESCRIPTION
**Pull request by kober32**
Resolved #52 - Added history API

### CODE REVIEW

File: library/src/androidTest/java/IntegrationTests.kt [hunk 1]
```diff
@@ -234,4 +239,35 @@ class IntegrationTests {
...
+        })
+
+        val operations = future.get(20, TimeUnit.SECONDS)
+        Assert.assertNotNull("Operations not retrieved" ,operations)
+        if (operations == null) {
+            return
+        }
+
+        Assert.assertEquals(operations.count(), 1 + operationsApproved + operationsRejected)
+        Assert.assertEquals(operations.count { it.status == OperationHistoryEntryStatus.APPROVED }, operationsApproved)
```
Comment (TomasKypta) [inline library/src/androidTest/java/IntegrationTests.kt:268]: Correct me if I'm wrong, but you seem to test against global variable that is set by a different unit test. This is not a good test design. Unit tests should be independent.

**Review by TomasKypta — COMMENTED** [review]
The code is ok. But the unit test is weird.

File: library/src/androidTest/java/IntegrationTests.kt (hunk 1 above)
Comment (kober32) [reply library/src/androidTest/java/IntegrationTests.kt]: You're right, the original idea was to build/test the defined lifecycle of the powerauth activation process, but it's not such a good idea.

I added a commit that makes every integration test autonomous.

**Review by TomasKypta — APPROVED** [review]
looks ok

**Pull request merged**